src/components/navigation/AuthNavigator.js

```diff
@@ -1,6 +1,19 @@
-import { createStackNavigator } from 'react-navigation-stack';
-import Login from '../Login';
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import NavigationConstants from 'components/navigation/NavigationConstants';
+import Login from 'components/Login';
 
-const AuthStack = createStackNavigator({ Login });
+const AuthNavigator = () => {
+  const Stack = createStackNavigator();
```

Comment:
This should be outside of your component, otherwise a new `Stack` will be created on every navigator render (which would break some optimizations of react-navigation when nesting navigators).